### ISSUE TITLE
FE web sockets

### ISSUE THREAD

**Issue by rrabinovitch** [HireUp-Turing/HireUp_frontend#2]
How might we use web sockets to allow for interaction between employers and applicants?

Resources to consider:
* https://getstream.io/
* https://socket.io/

**Comment by GreysonElkins** [comment]
This technology only supports one feature. Given that BE already has some stretches picked out, maybe treating that feature as an extension would be smartest? 

@rrabinovitch, @amykarnaze, @ErinUntermeyer, @gabichuelas

**Comment by ErinUntermeyer** [comment]
> This technology only supports one feature. Given that BE already has some stretches picked out, maybe treating that feature as an extension would be smartest?
> 
> @rrabinovitch, @amykarnaze, @ErinUntermeyer, @gabichuelas

I agree. I think that WebSockets is a wonderful extension. Let's focus on our other stretch techs first to build out MVP.
